doc/metrics.md

```diff
@@ -8,17 +8,15 @@ The number of CDI clone pods with high restart count. Type: Gauge.
 ### kubevirt_cdi_cr_ready
 CDI install ready. Type: Gauge.
 ### kubevirt_cdi_dataimportcron_outdated
-DataImportCron has an outdated import. Type: Gauge.
-### kubevirt_cdi_dataimportcron_outdated_aggregated
-Total count of outdated DataImportCron imports. Type: Gauge.
-### kubevirt_cdi_default_virt_storageclasses
-Number of default virt storage classes currently configured. Type: Gauge.
+DataImportCron has an outdated import. Type: GaugeVec.
+### kubevirt_cdi_datavolume_pending
+Number of DataVolumes pending for default storage class to be configured. Type: Gauge.
 ### kubevirt_cdi_import_pods_high_restart
 The number of CDI import pods with high restart count. Type: Gauge.
-### kubevirt_cdi_incomplete_storageprofiles
-Total number of incomplete and hence unusable StorageProfile. Type: Gauge.
 ### kubevirt_cdi_operator_up
 CDI operator status. Type: Gauge.
+### kubevirt_cdi_storageprofile_status
```

Comment:
@arnongilboa Please elaborate on this metric. I see from the alert that there are 2 labels: default and virtdefault. What do they mean? are there additional labels?